### ISSUE TITLE
piw-add

### ISSUE THREAD

**Issue by bennuttall** [piwheels/piwheels#286]
A `piw-add` command to add new packages and versions

- Add new package (skip, description)
- Add new version (skip, released)
- Unskip a package
- Unskip a version
- Yank/unyank a version
- Update project description
- Add a package alias

**Comment by waveform80** [comment]
Nearly done with the tests on this and once this is all covered I do think we should merge this, *however* there's quite a bit I don't particularly like here...

Specifically, it feels like we're shoe-horning a bunch of functions which should be calls in their own right into options on other operations for no other reason than "I don't want to add more callables". For example, it's entirely possible to yank a package and simultaneously remove all its builds ... which makes no sense. It also means the response messages from the master can conflate and make little sense. For example, if I yank a version the response message is 'YANKVER'. If I skip a version, the response is 'SKIPVER'. But if I yank *and* skip a version (which makes *some* sense) then the response is just 'YANKVER'.

Basically it feels like operations such as skip, yank, remove builds, unskip, unyank, etc. need to be callables in their own right. I don't like the idea of separate utilities for each either, but perhaps it's time we considered ditching `piw-import`, `piw-remove`, and the `piw-add` from this PR and making a `piw-manage` script with separate sub-commands? (e.g. import, add, delete, skip, unskip, yank, unyank?)

Anyway, that shouldn't get in the way of this landing -- it's something we can consider for a future revision.
